mysite/EStore/models.py

```diff
@@ -0,0 +1,84 @@
+from django.db import models
+
+# Create your models here.
+
+class Shops(models.Model):
+    shop_id = models.BigAutoField(primary_key = True)
```

Comment:
should be auto-generate, can remove this field. Only use it if using a function generating.